src/pytorch_lightning/trainer/trainer.py

```diff
@@ -297,9 +297,8 @@ def __init__(
 
             logger: Logger (or iterable collection of loggers) for experiment tracking. A ``True`` value uses
                 the default ``TensorBoardLogger``. ``False`` will disable logging. If multiple loggers are
-                provided and the `save_dir` property of that logger is not set, local files (checkpoints,
-                profiler traces, etc.) are saved in ``default_root_dir`` rather than in the ``log_dir`` of any
-                of the individual loggers.
+                provided, local files (checkpoints, profiler traces, etc.) are saved in the ``log_dir`` of
+                the first logger.
```

Comment:
can we add same info here, too? https://pytorch-lightning.readthedocs.io/en/stable/common/checkpointing_intermediate.html?highlight=checkpointing#where

Reply:
I have added some basic info https://github.com/Lightning-AI/lightning/pull/14325/commits/bff87a0d082d346da5d029e1dea7e30a89c670f9, let me know how to polish it.